package.json

```diff
@@ -6,8 +6,10 @@
   { "type"      : "git"
   , "url"       : "git://github.com/tim-smart/node-closure.git"
   }
+, "dependencies": {
+    "google-closure-compiler": "x"
```

Comment:
You are right.

Main issue would be between deployment of developers apps. This library doesn't have any google closure specific code, it just passes around arguments from the developer.

Might look into something like `peerDependencies`